bridgestyle/qgis/togeostyler.py

```diff
@@ -348,7 +348,7 @@ def processLabeling(layer, labeling, name="labeling", filter=None):
     # labels rendering scale-based visibility
     scale = None
     if settings.scaleVisibility:
```

Comment:
Since we are fixing this anyway, could you perhaps check if ``settings.scaleVisibility`` includes the same scales as ``layer.minimumScale()`` and ``layer.maximumScale()``? I suspect that it does, but I currently don't have time to check this.

If it does, I think it's fine if we simplify this to:

```suggestion
if hasattr(labeling, 'dependsOnScale') and labeling.dependsOnScale():
        scale = processRuleScale(labeling)
    elif layer.hasScaleBasedVisibility():
        scale = processRuleScale(layer)
```

... and ignore ``settings.scaleVisibility``.

Reply:
(do not simply commit this suggestion btw, I see it's indented wrong)